esp/scm/ws_dfu.ecm

```diff
@@ -901,6 +901,7 @@ ESPStruct DFUFileAccessInfo
 
 ESPenum DFUFileType : string
 {
+    Unset("Unset"),
```

Comment:
Could adding this at the start cause any problems?  Are these values persisted in dfu workunits?

Reply:
not persisted, but might cause problems for old client vs new actually..
Going to move to end.